app/globals.css

```
@@ -155,18 +155,18 @@
 }
 
 @layer utilities {
-  /* Stagger animations for children */
+  /* Stagger animations for children — Emil Kowalski: short delays, subtle motion */
   .stagger-children > * {
     @apply opacity-0;
-    animation: fadeInUp 0.8s ease-out forwards;
+    animation: fadeInUp 0.25s cubic-bezier(0.25, 0.1, 0.25, 1) forwards;
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)